src/Network/Network/help/New-AzApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration.md

```diff
@@ -0,0 +1,79 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://learn.microsoft.com/powershell/module/az.network/new-azapplicationgatewayfirewallpolicysettinglogscrubbingconfiguration
+schema: 2.0.0
+---
+
+# New-AzApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration
+
+## SYNOPSIS
+Creates a log scrubbing configuration for firewall policy
+
+## SYNTAX
+
+```
+New-AzApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration -State <String> -ScrubbingRule  <PSApplicationGatewayFirewallPolicySettingLogScrubbingRule[]> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration** creates a log scrubbing configuration for firewall policy.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+$logScrubbingRuleConfig = New-AzApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration -State Enabled -ScrubbingRule $logScrubbingRule1
```

Comment:
Could you add an example where a log scrubbing config has multiple scrubbing rules as well?

Reply:
There is one with two scurbbing rules in the policysetting help file.